modules/swffg-main.js

```diff
@@ -885,6 +887,19 @@ Hooks.once("ready", async () => {
       });
     }
   }
+
+  Hooks.on("refreshToken", (token) => {
+    /*
+    Used to render minion count
+    */
+    if (token?.actor?.type === "minion") {
+      drawMinionCount(token);
+    }
+    if (["character"].includes(token?.actor?.type)) {
```

Comment:
The `if` format switcheroo.

Reply:
haha, FWIW that is because I expect the one on line 898 to be expanded in the future, whereas 895 is static. fair point, though.